src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/RemoteVettingController.php

```diff
@@ -183,6 +183,7 @@ public function remoteVetMatchAction(Request $request, $processId)
 
             $form = $this->createForm(RemoteVetValidationType::class, $command)->handleRequest($request);
             if ($form->isSubmitted() && $form->isValid()) {
+                $this->logger->info('Finishing matching the remote vetting attributes');
```

Comment:
Perhaps add a final log entry with a report whether or not the matching was successful or not.

Reply:
Good suggestion, done